1.8/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos/s2i-base-centos7
+
+ENV NAME=golang \
+    VERSION=1.8 \
+    SCL_PATH=/opt/rh/go-toolset-7/
```

Comment:
This doesn't seem to be used. However, if it is defined for the case of being able to tell whether the go is SCL or not, I'd suggest to use `/opt/rh/go-toolset-7/root` at least, so it can be used as "root directory" in some paths.

Reply:
Will drop it. I thought that I will need it, but it turned out that I don't.